src/components/hooks/useMembers.js

```diff
@@ -35,7 +35,11 @@ export default function useMembers(destinationID, destinationType) {
       console.error(error.message);
     };
     const onMembers = (memberList) => {
-      setMembers([...memberList]);
+      if (Array.isArray(memberList)) {
```

Comment:
Why was this change needed?

Reply:
While committing was observing error for these lines